packages/core/src/drivers/db/models/estado.js

```diff
@@ -19,7 +19,7 @@ const EstadoSchema = {
       key: 'id',
     },
   },
-  estado: {
+  nombre: {
```

Comment:
Why did you change "estado" to "nombre"? If it's not a technical requirement, we should leave like "estado" is more expressive

Reply:
I think that if the table is estados, these are properties of the objec Estado, for example, name, description, location, and it is repetitive if I put estado as a property of estados. It is like a Persona has propertie nombre, and not a property called persona.